src/test/resources/indexListJsonString.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Replaced the old index fixtures with these updated ones.

Reply:
Do we usually have binary files that act as fixtures for our tests? From what I remember from Python-world, I would make a `setUp` function that'd write whatever fixtures I'd need for my tests, but then I'd also write a `tearDown` function that'd delete the files after the tests ran, just to avoid clogging up the repo.

Reply:
I'm not sure what the standard is generally, but I don't necessarily see an issue with keeping the files themselves around, especially given they're in the `/test/resources/` folder which I think is somewhat standard in Java projects. For this PR though I was mainly following what was being done previously for representing some of these request responses.

In my opinion it's a bit easier to grep what's in the JSON payload with things represented in a specific file rather than done programmatically as part of the test but that's just imo. I'd definitely be open to changing up how things are done.

Reply:
I think both approaches are fine, and in this case where it seems to static test configuration then keeping it around as a resource is totally fine. 

Sometimes it can be helpful/easier to generate test data and in those cases we definitely want to clean up afterwards. An example would be, let's say we need to have a JSON list of 10,000 fake users. It's like 5 lines of code to generate that vs. checking in an actual test file with 10,000 users, and managing that over time.

Reply:
Cool, sounds good!